app/_how-tos/throttle-apis-with-services-and-consumers.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Throttle APIs with different rate limits for Services and Consumers
+content_type: how_to
+
+related_resources:
+  - text: Rate Limiting
+    url: /rate-limiting/
+  - text: Rate Limiting with {{site.base_gateway}}
+    url: /gateway/rate-limiting/
+  - text: How to create rate limiting tiers with {{site.base_gateway}}
+    url:  /how-to/add-rate-limiting-tiers-with-kong-gateway/
+
+tier: enterprise
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+    - konnect
+
+min_version:
+  gateway: '3.9'
+
+plugins:
+  - rate-limiting
+  - service-protection
+
+entities: 
+  - service
+  - consumer
+  - plugin
+
+tags:
+    - rate-limiting
+
+tldr:
+    q: How do I throttle APIs to protect them from DDoS attacks while allowing multiple privileged consumers to access the Service with higher rate limits?
+    a: Configure the [Rate Limiting](/plugins/rate-limiting/) plugin on two consumers with `config.minute` set to 6. Configure the [Service Protection](/plugins/service-protection/) plugin with `config.window_size` set to 60 and `config.limit` set to 10. This setup will limit all requests on the Service to 10 per minute, even if the Consumers are sending requests simultaneously. 
+
+tools:
+    - deck
+
+prereqs:
+  entities:
+    services:
+        - example-service
+    routes:
+        - example-route
+
+cleanup:
+  inline:
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+---
+
+## 1. Create privileged Consumers
+
+In this how to, we'll be creating two Consumers that have different rate limits from the limits applied to the Service. These Consumers act as a type of "privileged" Consumer in this scenario that have higher rate limits, but they will still be prevented from exceeding the Service rate limits. The reason for this is to prevent DDoS type attacks on your APIs. 
```

Comment:
```suggestion
In this tutorial, we'll be creating two Consumers that have different rate limits from the limits applied to the Gateway Service. 
These Consumers act as a type of "privileged" Consumer in this scenario that have higher rate limits, but they will still be prevented from exceeding the Service rate limits. 
The reason for this is to prevent DDoS type attacks on your APIs. 
```
We've been using "tutorial" as a colloquial term for how-tos, I think it flows better in a sentence. It's currently used in the prereqs as well.